charts/apisix/values.yaml

```diff
@@ -268,6 +268,13 @@ extPlugin:
   enabled: false
   cmd: ["/path/to/apisix-plugin-runner/runner", "run"]
 
+wasmPlugins:
+  enabled: false
+  plugins:
```

Comment:
Let's set the `plugins` to `[]`, and comment the below `custom-plugin`. The `custom-plugin` should not be an option but an example.

Reply:
+1

Reply:
hello, got it will update it now